prestoadmin/util/presto_config.py

```diff
@@ -69,21 +69,17 @@ def from_file(config, config_path=None, config_host=None):
     def coordinator_config():
         config_path = os.path.join(REMOTE_CONF_DIR, CONFIG_PROPERTIES)
         config_host = env.roledefs['coordinator'][0]
-        try:
-            data = StringIO()
-            with settings(host_string='%s@%s' % (env.user, config_host)):
-                with hide('stderr', 'stdout'):
-                    temp_dir = run('mktemp -d /tmp/prestoadmin.XXXXXXXXXXXXXX')
-                try:
-                    get(config_path, data, use_sudo=True, temp_dir=temp_dir)
-                finally:
-                    run('rm -r %s' % temp_dir)
-
-            data.seek(0)
-            return PrestoConfig.from_file(data, config_path, config_host)
-        except:
-            _LOGGER.info('Could not find Presto config.')
-            return PrestoConfig(None, config_path, config_host)
+        data = StringIO()
+        with settings(host_string='%s@%s' % (env.user, config_host)):
+            with hide('stderr', 'stdout'):
+                temp_dir = run('mktemp -d /tmp/prestoadmin.XXXXXXXXXXXXXX')
+            try:
+                get(config_path, data, use_sudo=True, temp_dir=temp_dir)
+            finally:
+                run('rm -r %s' % temp_dir)
+
+        data.seek(0)
+        return PrestoConfig.from_file(data, config_path, config_host)
```

Comment:
What I meant is I think we should catch the exception here and call the error function with a nice message.

```
except Exception as e:
    error('Could not get coordinator config', exception=e)
```